@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -732,6 +732,19 @@ export interface SolidLineTechnique extends BaseTechnique, PolygonalTechnique {
      * Clip the line outside the tile if `true`.
      */
     clipping?: boolean;
+    /**
+     * Color of a line's outline in hexadecimal or CSS-style notation, for example: `"#e4e9ec"`,
+     * `"#fff"`, `"rgb(255, 0, 0)"`, or `"hsl(35, 11%, 88%)"`.
+     */
+    outlineColor?: string;
+    /**
+     * Width of a line's outline in `metricUnit`s for different zoom levels.
+     */
+    outlineWidth?: number;
+    /**
+     * The render order of the outline object created using this technique.
+     */
+    outlineRenderOrder?: number;
```

Comment:
> Why is this needed at all ?

Answered by myself, leaving it for others who may ask this question too ...

Actually, for roads all outlines should be in one range and all foregrounds have to be in distinct ranges we have to do it manually.

Reply:
I agree that specifying a secondary renderOrder is really weird, but it's the only current way to support this with the current API status (things might be different when a proper renderOrder management system is in place).